dhis-2/dhis-web-api/src/main/resources/openapi/TrackedEntitiesExportController.md

```diff
@@ -89,26 +112,58 @@ Get tracked entities using given organisation unit mode.
 
 ### `*.parameter.TrackedEntityRequestParams.program`
 
+Get tracked entity with tracked entity attribute and enrollment data from the specified program. The
+ownership to the given `program` will be used to determine access to the tracked entities.
+
 ### `*.parameter.TrackedEntityRequestParams.programStatus`
 
+Get a tracked entity that has at least one enrollment with the status specified.
+
+Valid options are:
+- `ACTIVE`
+- `COMPLETED`
+- `CANCELLED`
+
 ### `*.parameter.TrackedEntityRequestParams.followUp`
 
+Gets a tracked entities that has at least one enrollment that is marked with follow up.
+
 ### `*.parameter.TrackedEntityRequestParams.updatedAfter`
 
+Gets tracked entities that is updated after the given time. The update can be on the tracked entity
+or on one of the tracked entity attributes, enrollments or events for that tracked entity.
+
 ### `*.parameter.TrackedEntityRequestParams.updatedBefore`
 
+Gets tracked entities that is updated before the given time. The update can be on the tracked entity
+or on one of the tracked entity attributes, enrollments or events for that tracked entity.
+
 ### `*.parameter.TrackedEntityRequestParams.updatedWithin`
 
 ### `*.parameter.TrackedEntityRequestParams.enrollmentEnrolledAfter`
 
+Gets tracked entities that has at least one enrollment with an enrollment date after the date
+specified in `enrollmentEnrolledAfter`.
```

Comment:
if we write something like `after this date` we don't have to repeat the parameter name which will be right next to the description in the docs UI.